relay/client/stratoschain_events.go

```diff
@@ -197,6 +246,46 @@ func (m *MultiClient) CreateIndexingNodeMsgHandler() func(event coretypes.Result
 	}
 }
 
+func (m *MultiClient) UpdateIndexingNodeStakeMsgHandler() func(event coretypes.ResultEvent) {
+	return func(result coretypes.ResultEvent) {
+		networkAddressList := result.Events["update_indexing_node_stake.network_address"]
+		if len(networkAddressList) < 1 {
+			utils.ErrorLog("No network address was specified in the update_indexing_node_stake message from stratos-chain")
+			return
+		}
+
+		p2pAddress, err := types.BechToAddress(networkAddressList[0])
+		if err != nil {
+			utils.ErrorLog("Error when trying to convert P2P address to bytes", err)
+			return
+		}
+		p2pAddressString, err := p2pAddress.ToBech(setting.Config.BlockchainInfo.P2PAddressPrefix)
+		if err != nil {
+			utils.ErrorLog("Error when trying to convert P2P address to bech32", err)
+			return
+		}
```

Comment:
From line 252 to 266, these lines are repeated in almost every function. We may extract it as a single validator function to avoid code replication.

Reply:
Yep agree. This can be extracted as a func. We could do it in later PR

Reply:
New ticket QB-720 created and will be fixed later